api/src/opentrons/protocol_api/contexts.py

```diff
@@ -362,66 +380,149 @@ def mix(self,
             repetitions: int = 1,
             volume: float = None,
             location: Well = None,
-            rate: float = 1.0):
+            rate: float = 1.0) -> 'InstrumentContext':
         raise NotImplementedError
 
-    def blow_out(self, location: Well = None):
+    def blow_out(self, location: Well = None) -> 'InstrumentContext':
         raise NotImplementedError
 
     def touch_tip(self,
                   location: Well = None,
                   radius: float = 1.0,
                   v_offset: float = -1.0,
-                  speed: float = 60.0):
+                  speed: float = 60.0) -> 'InstrumentContext':
         raise NotImplementedError
 
     def air_gap(self,
                 volume: float = None,
-                height: float = None):
+                height: float = None) -> 'InstrumentContext':
         raise NotImplementedError
 
-    def return_tip(self, home_after: bool = True):
+    def return_tip(self) -> 'InstrumentContext':
         raise NotImplementedError
 
-    def pick_up_tip(self, location: Well = None,
+    def pick_up_tip(self, location: types.Location = None,
                     presses: int = 3,
-                    increment: int = 1):
-        raise NotImplementedError
+                    increment: float = 1.0) -> 'InstrumentContext':
+        """`
+        Pick up a tip for the Pipette to run liquid-handling commands with
+
+        A tip can be manually set by passing a `location`. If no location
+        is passed, the Pipette will pick up the next available tip in
+        its `tip_racks` list (see :any:`Pipette`)
+
+        :param location: The labware or well from which to pick up a tip.
```

Comment:
```suggestion
        :param location: The location from which to pick up a tip.
```

It's a little more generically worded just to make sure that people don't get upset when they can't pass a `Labware` or `Well` directly